contracts/ERC20.sol

```diff
@@ -2,25 +2,70 @@
 pragma solidity ^0.8.0;
 
 import "./IERC20.sol";
+import "./IERC20Metadata.sol";
+
+contract ERC20 is IERC20, IERC20Metadata {
+  uint256 public _totalSupply;
+  string public _name;
+  string public _symbol;
+  uint8 public _decimals;
+  mapping(address => uint256) public ownership;
```

Comment:
Same for this as well, since you have already has balanceOf() as the preferred way to access it